sklearn/lda.py

```diff
@@ -53,7 +53,7 @@ def _cov(X, shrinkage=None):
         if shrinkage == 'auto':
             sc = StandardScaler()  # standardize features
             X = sc.fit_transform(X)
-            s = sc.std_ * ledoit_wolf(X)[0] * sc.std_  # scale back
+            s = sc.std_ * ledoit_wolf(X)[0] * sc.std_[np.newaxis].T  # rescale
```

Comment:
I would write:

s = sc.std_[:, np.newaxis] * ledoit_wolf(X)[0] * sc.std_sc.std_[np.newaxis, :]

to be super explicit.